maker-frontend/src/MakerApp.tsx

```diff
@@ -49,12 +49,13 @@ export default function App() {
 
```

Comment:
I'm confused about this commit. Is this logic maybe a bit outdated?

Reply:
yeah, this was the starting point from @rishflab's PR. I have created 2 separate spread values in one of the further commits - it doesn't really matter much in the maker UI apart from actually seeing 2 different prices IMHO, as it's more of an example and dev testing env